### ISSUE TITLE
Feature Request - Event Report / Export to file

### ISSUE THREAD

**Issue by andyaz** [xspanger3770/GlobalQuake#188]
I would like to have a file created when GQ Client sees a new event including on the GQ Client when talking to the a server. A simple comma delimited file with the Event time, Event Region, Event Magnitude, and Event MMI.

I could use this file with an external program to tail the file and see the new event and use it to generate a voice announcement or other external information for use on my livestream.

**Comment by DecryptingElectrons** [comment]
I like this idea as it is a simple output. Right now i am working on a FDSNWS event server that will run inside GQ, that a pr will made for soon

**Comment by richthedee** [comment]
Hello, is this feature still needed ? Need to fix a issue for my school assigment

**Comment by andyaz** [comment]
> Hello, is this feature still needed ? Need to fix a issue for my school assigment

Yes, I could still use this ability, particularly on the client side. I want to be able to use something like NodeRed to check the file for new event to generate TTS to speak the event on my livestream. Since the stream runs on my client computer and not my server being able to have a consistently formatted CSV file for external programs on both the client and server to use would be most helpful. It could keep say the last 100 events containing the event time, event quality, event region, event magnitude, and event MMI. Maybe trigger it to write when the event has reached say a quality C or better.

**Comment by richthedee** [comment]
Okay, I will look into it, hopefully this month. Thanks for quick response.

**Comment by andyaz** [comment]
Any word in the dev side of when we may see a release happen?

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by xspanger3770** [comment]
Feature request now tracked internally.
https://github.com/xspanger3770/GlobalQuake/tree/announcement
